src/start_server.ts

```diff
@@ -143,6 +150,24 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxyRoot) {
+    const httpProxy = require('http-proxy');
```

Comment:
Put all of our module loading at the top of the file. You can do `import httpProxy = require('http-proxy')` in TS.